### ISSUE TITLE
Calculation of domain area assumes network is cubic-ish

### ISSUE THREAD

**Issue by jgostick** [PMEAL/OpenPNM#1194]
The _get_domain_area function uses convex hull calculations with the implicit assumption that the hull is 2D.  This must be fixed STAT!

**Comment by jgostick** [comment]
Here is a snippet that illustrates the problem:

```
import openpnm as op
ws = op.Workspace()
proj = ws.new_project()

pn = op.network.Delaunay(num_points=100, shape=[1, 1, 1])

Ps = pn.find_neighbor_pores(pores=pn.pores(['left']))
pn['pore.left'][Ps] = True

Ps = pn.find_neighbor_pores(pores=pn.pores(['right']))
pn['pore.right'][Ps] = True

op.topotools.trim(pn, pores=pn.pores('boundary'))

geo = op.geometry.StickAndBall(network=pn, pores=pn.Ps, throats=pn.Ts)
water = op.phases.Water(network=pn, name='h2o')
phys_water = op.physics.Standard(network=pn, phase=water, geometry=geo)

perm = op.algorithms.StokesFlow(network=pn)
perm.setup(phase=water)
perm.set_value_BC(pores=pn.pores('right'), values=0)
perm.set_value_BC(pores=pn.pores('left'), values=101325)
perm.run()
perm._get_domain_area()
```

It gives an area around 0.1 instead of 1.0

**Comment by jgostick** [comment]
Fixed in PR #1200
